src/app/order/OrderStatus.tsx

```diff
@@ -11,27 +11,32 @@ export interface OrderStatusProps {
     order: Order;
 }
 
+const checkoutcomSEPAMethod = 'SEPA Direct Debit (via Checkout.com)';
```

Comment:
As this is also a constant config value, we can use the same all capitals, snake case as `TEXT_ONLY_METHODS` to better convey its purpose.